services/core-api/app/api/now_applications/resources/now_application_status_resource.py

```diff
@@ -63,6 +63,7 @@ def put(self, application_guid):
         auth_end_date = data.get('auth_end_date')
         status_reason = data.get('status_reason')
         description = data.get('description')
+        decision_by_user_date = data.get('decision_by_user_date', None)
```

Comment:
As far as I can tell this isnt getting used.

Reply:
removed.